Form/Type/DocumentType.php

```diff
@@ -20,9 +20,24 @@
 use Doctrine\Bundle\PHPCRBundle\Form\ChoiceList\PhpcrOdmQueryBuilderLoader;
 use Symfony\Bridge\Doctrine\Form\ChoiceList\EntityLoaderInterface;
 use Symfony\Bridge\Doctrine\Form\Type\DoctrineType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
 class DocumentType extends DoctrineType
 {
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        parent::setDefaultOptions($resolver);
+        $type = $this;
+        $loader = function (Options $options) use ($type) {
+            $qb = $options['query_builder'] ?: $options['em']->createQueryBuilder()->fromDocument($options['class'], 'a');
+
+            return $type->getLoader($options['em'], $qb, $options['class']);
```

Comment:
okay, i solved that mistery. @guilhermeblanco fixed this 10 day ago! https://github.com/symfony/symfony/commit/529d99c567a532ae944682008fa0fc7db179db98 - that commit is not yet in a 2.3 release.

should we still do this to work with older versions?